code/game/machinery/doors/door.dm

```diff
@@ -270,6 +270,7 @@
 	return src.attack_hand(user)
 
 /obj/machinery/door/attack_hand(mob/user as mob)
+	src.add_fingerprint(user)
```

Comment:
Ditto.

Reply:
It seriously doesn't work if it's just add_fingerprint. Really, pull the master and just add that in as add_fingerprint and it won't work. It only works with .src - It's some fucky door thing or something idk.